packages/react-ui-core/src/Photo/Photo.js

```diff
@@ -0,0 +1,62 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+
+@themed(['Photo'], {
+  pure: true,
+})
+
+export default class Photo extends PureComponent {
+  static propTypes = {
+    url: PropTypes.string,
+    fallbackUrl: PropTypes.string,
+    theme: PropTypes.object,
+    className: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+  }
+
+  static getDerivedStateFromProps(props, state) {
+    if (!state.url || (state.url !== props.url)) {
+      return {
+        url: props.url,
+        error: false,
+      }
+    }
+
+    return null
+  }
+
+  fallback() {
+    this.setState({ error: true })
+  }
+
+  render() {
+    const {
+      theme,
+      className,
+      url: _,
```

Comment:
interesting, is this so that its omitted from rest?

Reply:
Yep 😄 , just renaming it to `_`